api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -393,7 +393,6 @@ public void test_updateUserWithRetries_dua_unbypassed_aar_wrong_version_noncompl
   public void test_updateUserWithRetries_dua_unbypassed_aar_expired_noncompliant() {
     testUnregistration(
         user -> {
-          user.setDataUseAgreementBypassTime(null);
```

Comment:
all test removals like this have `accessModuleService` equivalents, like on line 398 here.